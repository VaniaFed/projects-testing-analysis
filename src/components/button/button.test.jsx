import renderer from 'react-test-renderer';
import { Button } from './index';

describe('<Button />', () => {
    const tree = renderer.create(<Button text="Hello, React!"/>).toJSON();
    it('should be rendered with the text "Hello, React!"', () => {
        expect(tree).toMatchSnapshot();
    });
});
