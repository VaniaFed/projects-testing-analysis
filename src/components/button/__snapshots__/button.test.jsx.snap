// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should be rendered with text "Hello, React!" 1`] = `
<button
  className="button"
  data-button="search-page__button-search"
  type="button"
>
  Hello, React!
</button>
`;
